diff --git a/lua/options.lua b/lua/options.lua
index b797506..fbd6b75 100644
--- a/lua/options.lua
+++ b/lua/options.lua
@@ -242,8 +242,8 @@ opt.undodir = (os.getenv 'HOME' or '') .. '/.local/share/nvim'
 -- [[ Setting vim cmds ]]
 vim.cmd ':set display-=msgsep'
 -- vim.cmd ':set display-=lastline' -- No Line on left
-vim.cmd ':set nomore'
--- vim.cmd ':set more'
+-- vim.cmd ':set nomore'
+vim.cmd ':set more'
 -- vim.cmd ':set lz' -- Lazy Redraw
 -- vim.cmd ':set ttyfast' -- Lazy Redraw
 vim.cmd [[ :set iskeyword-=- ]]
diff --git a/lua/plugins/nvim-tree.lua b/lua/plugins/nvim-tree.lua
index 03fe3c7..7442566 100644
--- a/lua/plugins/nvim-tree.lua
+++ b/lua/plugins/nvim-tree.lua
@@ -354,7 +354,7 @@ return {
       size = {
         enable = true,
         column_width = 8,
-        width_cutoff = 12,
+        width_cutoff = 18,
         show_folder_size = not OnWsl(),
         format_unit = 'single',
       },
